### ISSUE TITLE
Ethernet Not Working

### ISSUE THREAD

**Issue by LouA1** [HelmSecure/armbian-images#19]
After flashing the helm my NIC port is no longer working.  Lights flash but I never get a green link light flashing.  Is there a factory Img to flash to return Helm V2A to factory defaults.  All indications in terminal appear to suggest the flash took.  I tried an external USB C NIC, but still no go.  Since initially installing Helm as an email service I had my router assign it a static IP, so I know which IP address it should be asssigned.  Any suggestions?

**Comment by kkanatcit** [comment]
I didn’t have this problem, but when I’ve had issues, I have been able to reflash the unit and restored it to the fresh Armbian image.  I’ve done this 4-5 times, once after my config changes resulted in kernel panic that prevented boot, and multiple times after the Helm froze up trying to format the 1TB SSD. I ended up having to take the SSD out and format it elsewhere and then was able to mount it.
Separately, it wouldn’t hurt to check your network to see if it picked up a different IP via DHCP somehow... good luck.

**Comment by dsigurds** [comment]
> After flashing the helm my NIC port is no longer working. Lights flash but I never get a green link light flashing. Is there a factory Img to flash to return Helm V2A to factory defaults. All indications in terminal appear to suggest the flash took. I tried an external USB C NIC, but still no go. Since initially installing Helm as an email service I had my router assign it a static IP, so I know which IP address it should be asssigned. Any suggestions?

Ethernet is something that we've tested quite a bit and haven't had issues with it.  It's possible that the ethernet MAC changed and so the IP reservation that you made is no longer valid.

**Comment by LouA1** [comment]
Dirk,Turns out, that’s exactly what happened.  It has a new MAC address. Thanks.Regards,LouSent from my iPadOn Feb 3, 2023, at 11:21 PM, Dirk Sigurdson ***@***.***> wrote:﻿

After flashing the helm my NIC port is no longer working. Lights flash but I never get a green link light flashing. Is there a factory Img to flash to return Helm V2A to factory defaults. All indications in terminal appear to suggest the flash took. I tried an external USB C NIC, but still no go. Since initially installing Helm as an email service I had my router assign it a static IP, so I know which IP address it should be asssigned. Any suggestions?

Ethernet is something that we've tested quite a bit and haven't had issues with it.  It's possible that the ethernet MAC changed and so the IP reservation that you made is no longer valid.
